src/planner/linux/steam_deck.rs

```diff
@@ -188,7 +189,7 @@ impl Planner for SteamDeck {
             Type=oneshot\n\
             RemainAfterExit=yes\n\
             ExecStart=/usr/bin/systemctl daemon-reload\n\
-            ExecStart=/usr/bin/systemctl restart --no-block sockets.target timers.target multi-user.target\n\
+            ExecStart=/usr/bin/systemctl start --no-block sockets.target timers.target multi-user.target\n\
```

Comment:
This is the load bearing line of the fix, the above line is relatively unrelated.